docroot/themes/humsci/humsci_basic/tests/sass-specs/sass-tests.scss

```diff
@@ -3,13 +3,13 @@ $hb-default-theme: 'colorful';
 $hb-current-theme: $hb-default-theme;
 
 @import 'true';
-@import '../../src/scss/main.scss';
+@import '../../src/scss/partials/main.scss';
 
 // Import all test partials here
 @include describe('Sass Unit Tests') {
   @import 'tools/mixins.themes';
-  @import 'tools/mixins.color-pairings';
-  @import 'tools/functions.color-pairings';
+  // @import 'tools/mixins.color-pairings';
```

Comment:
Not sure where these files went during the modular sass refactor. Leaving these commented out for now.